spark-doris-connector/src/main/scala/org/apache/doris/spark/sql/DorisSourceProvider.scala

```diff
@@ -115,7 +120,7 @@ private[sql] class DorisSourceProvider extends DataSourceRegister
 
           if (!rowsBuffer.isEmpty) {
             logger.warn("Data that failed to load : " + dorisStreamLoader.listToString(rowsBuffer))
```

Comment:
It is recommended to remove these two logs by the way [dorisStreamLoader.listToString], where all data will be printed to the log.

Reply:
done

Reply:
good